templates/items/consumable.hbs

```diff
@@ -2,11 +2,15 @@
 
     {{!-- Item Sheet Header --}}
     <header class="sheet-header flexrow">
-        <img class="profile" src="{{item.img}}" title="{{item.name}}" data-edit="img"/>
+        <img class="profile" src="{{item.img}}" title="{{displayName}}" data-edit="img"/>
 
         <div class="header-details flexrow">
             <h1 class="charname">
+                {{#if system.identified}}
                 <input name="name" type="text" value="{{item.name}}" placeholder="{{ localize 'DND5E.ItemName' }}"/>
+                {{else}}
+                <input disabled name="ignored" type="text" value="{{ localize (dnd5e-unidentifiedName item) }}"/>
```

Comment:
Disabled inputs are automatically ignored by form processing, so you don't need to worry about the name here.

```suggestion
                <input disabled type="text" value="{{ localize (dnd5e-unidentifiedName item) }}">
```